src/axiosFunctions.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+const ncNews = axios.create({
+  baseURL: "https://news-board-e14n.onrender.com",
+});
+
+export const getAllArticles = () => {
+  return ncNews.get("/api/articles").then(({ data }) => {
+    return data;
+  });
+};
+
+export const getArticle = (article_id) => {
+  return ncNews.get(`/api/articles/${article_id}`).then(({ data }) => {
+    return data;
+  });
+};
+
+export const timeConverter = (unformatted) => {
```

Comment:
This could and should really be in maybe a utils.js file and save this file for pure api/axios requests.